LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Just curious on the licensing; do we have any ability in relicensing this as GPL3? Not sure what Cosmos SDK and Tendermint are licensed as.

Reply:
sure, I just copied it from Ethermint since this is the license that usually the ICF enforces on the Cosmos projects. Do you want to open a PR with the new license?

Reply:
Will do—I'll double check on our side first :).